published/009-2016-March-28.md

```diff
@@ -0,0 +1,157 @@
+
+#### Dictionary support for the zlib JavaScript Implementation, [`pako`](https://github.com/nodeca/pako)
+
+One of the significant contributions made this week was the addition of 'dictionary' support for `zlib` JavaScript implementation, `pako`. With this contribution, we are able to have a complete implementation of SPDY 3.1's framing layer running in the browser, the default stream muxing library used in IPFS. You can find more about this contribution in the following issue and PR discussions:
+
+- https://github.com/nodeca/pako/issues/69
+- https://github.com/diasdavid/js-libp2p-spdy/pull/6
+- https://github.com/nodeca/pako/pull/77
+
+### go-ipfs
+- @whyrusleeping wrote a tool to move content from 0.4.0 to 0.3.11 (https://github.com/whyrusleeping/levart-emit). He also discovered a file descriptor leak bug in utp causing connectivity issues, and began work on datastore performance improvements.
+
+#### `jsipfs object` cli and http-api endpoints are complete
+
+Now you can use `jsipfs object` in the same way you would use `ipfs object`. Big thanks to [Francisco Dias](https://github.com/xicombd) for leading the last miles of this goal. The complete track of the development can be found at [github.com/ipfs/js-ipfs/issues/58](https://github.com/ipfs/js-ipfs/issues/58).
+
+### Nginx metrics
+
+Our internal metrics dashboard didn't previously have HTTP request/response metrics from nginx's point of view, but only from IPFS's and multireq's point of view. (Multireq is our v04x/v03x multiplexing proxy). Nginx itself provides finegrained metrics only through their commercial subscriptions. We're now using [mtail](https://github.com/google/mtail) to parse metrics from nginx access logs and expose them to [Prometheus](https://prometheus.io). @lgierth will also contribute the `nginx.mtail` program upstream with mtail.
+
+## Community
+
+### Meeting with NYC Mesh
+
+@jbenet and @lgierth met with the fine folks of [nycmesh.net](https://nycmesh.net). For the past two years they've been building a community Wifi network in New York City. We had lots of great conversation about wireless mesh networking and IPFS. If you live in NYC, you should come attend their [meetups](http://www.meetup.com/nycmesh/)!
+
+### Lisbon Research and Development
+
+[The IPFS Lisbon community had their second "Research & Development Meetup"](http://www.meetup.com/ipfs-lisbon-meetup/events/229807984/), hosted by Uniplaces (https://www.uniplaces.com). The focus was "The Distributed Web" and "Machine Learning + Artificial Intelligence for Recommender Algorithms", with talks by [David Dias](https://github.com/diasdavid) and [João Ascensão](https://github.com/jtascensao), respectively. If you are around Lisbon, make sure to join http://www.meetup.com/ipfs-lisbon-meetup to get notified about the next one.
+
+- Good photos: http://ipfs.io/ipfs/Qmd53hsuwkCcrhD1KixBwGMm4tCoLuQvpoW5HWRSqADCrW (feel free to apply filters to improve the colors)
```

Comment:
ahaha, pic one or two to give color to the weekly. Do not post it with the note I left you :P xD